includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab;
+
+use WP_Error;
```

Comment:
Let's avoid using namespaces for now, for a similar reason that I think we should avoid the class based approach: Unless there's a clear benefit, it's preferable to not introduce new coding patterns.

Reply:
This class has been dissolved entirely now @felixarntz